javascript-forms/form-events.js

```diff
@@ -0,0 +1,28 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('event.target.name:', event.target.name);
+  console.log('event.target.value:', event.target.value);
+}
+var $name = document.querySelector('input');
+var $email = document.querySelectorAll('input')[1];
```

Comment:
This seems dangerous, what if the order of the inputs was reversed? You should query the element using information specific to that element.